skyportal/handlers/api/gcn.py

```diff
@@ -577,3 +579,17 @@ def delete(self, dateobs, localization_name):
         self.verify_and_commit()
 
         return self.success()
+
+
+class GcnNoticeTypesHandler(BaseHandler):
+    @auth_or_token
+    def get(self):
+        '''
+        This method return the list of notices from the config file
+        ---
+        description: Returns the list of notices
+        tags:
+          - gcn_notices
+        '''
+        notices = cfg['gcn_notice_types']
+        return self.success(data=notices)
```

Comment:
I think you may be able to get this without using a separate handler. Look at `static/js/ducks/config.js` and `skyportal/handlers/api/config_handler.py` .

Reply:
Oh that's great ! I've completely missed that config_handler. I'll try using that instead.

Reply:
Done.